l10n_be_mis_reports/script/generate_xml.py

```diff
@@ -0,0 +1,465 @@
+import json
+import logging
+import os
+import re
+from xml.dom import minidom
+
+import click
+
+# CONST
+MIS_REPORT_MODEL = "mis.report"
+MIS_REPORT_KPI_MODEL = "mis.report.kpi"
+BS_REPORT_XMLID = "mis_report_bs"
+PL_REPORT_XMLID = "mis_report_pl"
+SPECIALSTYLES = ["9900", "9901", "9902", "9903", "9904"]
+PATH = "../data/"
+
+logging.basicConfig(level=logging.INFO)
+_logger = logging.getLogger(__name__)
+
+
+@click.command()
+@click.option("--author", help="name of the copyright author")
+@click.option("--year", help="date of the copyright")
+@click.argument("data-file", type=click.File())
+@click.argument("calc-file", type=click.File())
+def generate_xml(author, year, data_file, calc_file):
+    """This script generates two xml documents for the balance sheet
+    and the profit and loss table from DATA_FILE and CALC_FILE
+
+    DATA_FILE is a json file where all the data of the report is stored
+
+    CALC_FILE is a json file where accounts' calculations are stored
+    """
+    try:
+        os.mkdir(PATH)
+        _logger.info(f"{PATH} created")
+    except FileExistsError:
+        _logger.info(f"{PATH} already exists")
+
+    try:
+        json_data_file = json.load(data_file)
+        json_calc_file = json.load(calc_file)
```

Comment:
These are not files but dictionaries. 

```suggestion
        json_data_dict = json.load(data_file)
        json_calc_dict = json.load(calc_file)
```